wayland-sys/src/cursor.rs

```diff
@@ -53,7 +53,7 @@ lazy_static::lazy_static!(
         for ver in &versions {
             match WaylandCursor::open(ver) {
```

Comment:
This call should be unsafe, but the CI didn't catch it because the `cursor` feature of Wayland-sys is no longer used.